modules/nextflow.config

```diff
@@ -0,0 +1,39 @@
+// Global default params, used in configs
+params {
```

Comment:
Could you please merge this file with the main `nextflow.config` at the root of this repository?